<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HATI-SDG00096" failures="0" tests="3" name="OPDTest.Reschedule_New_Appointment_against_doctor" time="259.663" errors="0" timestamp="2023-12-06T10:54:58 IST" skipped="0">
  <testcase classname="OPDTest.Reschedule_New_Appointment_against_doctor" name="TC_01_verify_That_User_Is_Able_To_Login_MedcareApplication" time="10.082"/>
  <system-out/>
  <testcase classname="OPDTest.Reschedule_New_Appointment_against_doctor" name="TC_02_Verify_that_user_is_able_to_book_the_New_appointment" time="107.048"/>
  <system-out/>
  <testcase classname="OPDTest.Reschedule_New_Appointment_against_doctor" name="TC_03_Verify_That_User_Is_Able_Reshcedule_the_appointment" time="142.533"/>
  <system-out/>
</testsuite> <!-- OPDTest.Reschedule_New_Appointment_against_doctor -->
